client/blocks/reader-full-post/_content.scss

```diff
@@ -58,6 +58,30 @@
 
 .is-reader-page .reader-full-post__story-content {
 	padding-top: 0 !important;
+
+	.wp-block-gallery .wp-block-image {
```

Comment:
You probably need to add classes for images added with the classic editor and/or with the Classic block as well?

Reply:
Hmm... We probably are going to have to deal with images in a classic block the same way we're dealing with images with a caption (by injecting a container element around the image). There is nothing here that tells me this image is in a classic block, and I can't attach a box-shadow to a generic `p` tag as that will surely lead to edge case issues down the line. cc @eoigal 

Here's the code for an image in a classic block:

![CleanShot 2022-10-14 at 10 54 56](https://user-images.githubusercontent.com/5634774/195878035-7ee6374e-9259-4d67-a61c-28dd4050fc7f.png)

Reply:
For testing: `/read/feeds/133479798/posts/4320194408`